docs/source/performance/apq.md

```diff
@@ -182,171 +182,9 @@ How exactly this works depends on exactly which CDN you chose. Configure your CD
-| Memcached  | `MemcachedCache`  | [`apollo-server-cache-memcached`](https://npm.im/apollo-server-cache-memcached)  |
-| Redis (single instance or Sentinel)  | `RedisCache`  | [`apollo-server-cache-redis`](https://npm.im/apollo-server-cache-redis)  |
-| Redis Cluster | `RedisClusterCache`| [`apollo-server-cache-redis`](https://npm.im/apollo-server-cache-redis)|
-
-Examples for supported data stores are provided below.
-
-### Memcached
-
-```shell
-$ npm install apollo-server-cache-memcached
-```
-
-```javascript
-const { MemcachedCache } = require('apollo-server-cache-memcached');
-const { ApolloServer } = require('apollo-server');
-
-const server = new ApolloServer({
-  typeDefs,
-  resolvers,
-  csrfPrevention: true,
-  cache: 'bounded',
-  // highlight-start
-  persistedQueries: {
-    cache: new MemcachedCache(
-      ['memcached-1.local', 'memcached-2.local', 'memcached-3.local'],
-      { retries: 10, retry: 10000 }, // Options
-    ),
-  },
-  // highlight-end
-});
-```
-
-### Redis (single instance)
-
-```shell
-$ npm install apollo-server-cache-redis ioredis
-```
-
-```javascript
-const { BaseRedisCache } = require('apollo-server-cache-redis');
-const Redis = require('ioredis');
-
-const server = new ApolloServer({
-  typeDefs,
-  resolvers,
-  csrfPrevention: true,
-  cache: 'bounded',
-  // highlight-start
-  persistedQueries: {
-    cache: new BaseRedisCache({
-      client: new Redis({
-        host: 'redis-server',
-      }),
-    }),
-  },
-  // highlight-end
-});
-```
-
-### Redis (Sentinel)
-
-```shell
-$ npm install apollo-server-cache-redis ioredis
-```
-
-```javascript
-const { BaseRedisCache } = require('apollo-server-cache-redis');
-const Redis = require('ioredis');
-
-const server = new ApolloServer({
-  typeDefs,
-  resolvers,
-  csrfPrevention: true,
-  cache: 'bounded',
-  // highlight-start
-  persistedQueries: {
-    cache: new BaseRedisCache({
-      client: new Redis({
-        sentinels: [{
-          host: 'sentinel-host-01',
-          port: 26379
-         }],
-        password: 'my_password',
-        name: 'service_name',
-      }),
-    }),
-  },
-  // highlight-end
-});
-```
-
-### Redis Cluster
-
-```shell
-$ npm install apollo-server-cache-redis ioredis
-```
-
-```javascript
-const { BaseRedisCache } = require('apollo-server-cache-redis');
-const Redis = require('ioredis');
-
-const server = new ApolloServer({
-  typeDefs,
-  resolvers,
-  csrfPrevention: true,
-  cache: 'bounded',
-  // highlight-start
-  persistedQueries: {
-    cache: new BaseRedisCache({
-      // Note that this uses the "noMgetClient" option rather than "client",
-      // which avoids using the mget command which doesn't work in cluster mode.
-      noMgetClient: new Redis.Cluster(
-        [{
-          host: 'redis-node-01-host',
-        }],
-        {
-          // Other Redis cluster client options
-        }
-      ),
-    }),
-  },
-  // highlight-end
-});
-```
-
-## Adjusting cache time-to-live (TTL)
```

Comment:
Did you mean to remove this section?

Reply:
Whoops, definitely not! 97483f6d915a1537f0c6e27e3e86d500ea5ba72b